docs/UserGuide.md

```diff
@@ -136,25 +138,38 @@ Example:
 
 ### Sorting contacts : `sort`
 
-Sort the list of contacts displayed by certain parameter(s).
+Sort the list of contacts displayed by 1 or more parameter(s) chosen by you.
+
+How *names (n/)*, *emails (e/)*, *addresses (a/ )* are sorted:
+* In alphabetical order. (e.g. `Al` comes before `Alfred` which comes before `Brad`)
+* Case differences are ignored. (e.g. `Al`, `al`, `AL`, and `aL` are identical when it comes to sorting)
+
+How *phone numbers (p/)* are sorted: 
+* In increasing numerical order. (e.g. `123` comes before `125` which comes before `1234`)
+
+How *tags (t/TAG)* are sorted:
+* Contacts with the *TAG* you specified will come before contacts without the *TAG*.
```

Comment:
Just to clarify. Does this mean that those without the right `TAG` is in default sorting arrangement?

Reply:
yea